src/main/java/org/icgc/argo/program_service/grpc/ProgramServiceImpl.java

```diff
@@ -54,11 +54,12 @@ public ProgramServiceImpl(@NonNull ProgramService programService, @NonNull Progr
   }
 
 
+  //TODO: need better error response. If a duplicate program is created, get "2 UNKNOWN" error. Should atleast have a message in it
   @Override
   @EgoAuth(typesAllowed = {"ADMIN"})
   public void createProgram(CreateProgramRequest request, StreamObserver<CreateProgramResponse> responseObserver) {
     val program = request.getProgram();
-    val entity = programService.createProgram(program);
+    val entity = programService.createProgram(program, request.getInitialAdminEmailsList());
```

Comment:
emailsList or emailList?

Reply:
This is the byproduct of using plurals in repeated field definitions.
PLURAL: `repeated string emails` will be represented in java as `List<String> getEmailsList()`
SINGLULAR:  `repeated string email` will be represented in java as `List<String> getEmailList()`

All though the [google protobuf style guidlines](https://developers.google.com/protocol-buffers/docs/style#repeated-fields) suggests using PLURAL form, there is however another argument that SINGLAR makes sense since the keyword `repeated` indicates its a list of that element. 

I think we should stick with PLURAL, because in ticket #41, i will be extending the Mapstruct SPI to map protobuf generated methods ending in List to the corresponding Bean definition for a collection. For example, mapping getPoliciesList from the proto to getPolicies on the bean. If i was using singular, i would have to add extra logic on when to add `ies` and just `s` as a suffix (i.e getPolicyList in the proto to getPolicies in the bean). In the case of plural, i just have to drop the `List` portion of the method name, as the model is already defined with the plural name. Lombok unfortunatly does now have a way to specify the suffix for collections.